rails/etsy_product_catalog/app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CategoriesController < ApplicationController
+	before_action :find_category, only: [:show]
+
+	def index
+		@categories = Category.all
+	end
+
+	def show
+
+	end
+end
+
+
+private
+
+	def find_category
+		@Category = Category.find(params[:id])
```

Comment:
The capitalization will bite you here - it doesn't match what's in the view template for show.